src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -415,6 +444,23 @@ internal Notification CreateNotification(NotificationVersionOne v1Notification)
             return notification;
         }
 
+        internal Replacement CreateReplacement(ReplacementVersionOne v1Replacement)
+        {
+            Replacement replacement = null;
+
+            if (v1Replacement != null)
+            {
+                replacement = new Replacement
+                {
+                    DeletedLength = v1Replacement.DeletedLength,
+                    InsertedBytes = v1Replacement.InsertedBytes,
+                    Offset = v1Replacement.Offset
```

Comment:
In the next PR, this is the only part of the converter logic that will need to change.